openmc/universe.py

```diff
@@ -367,12 +370,23 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
         if basis == 'xy':
             x, y = 0, 1
             xlabel, ylabel = 'x [cm]', 'y [cm]'
+                
         elif basis == 'yz':
             x, y = 1, 2
             xlabel, ylabel = 'y [cm]', 'z [cm]'
+
         elif basis == 'xz':
             x, y = 0, 2
             xlabel, ylabel = 'x [cm]', 'z [cm]'
+
+        if width is None:
+            x_width = self.bounding_box.width(basis[0])
+            y_width = self.bounding_box.width(basis[1])
+            width = (x_width, y_width)
+
+        if origin is None:
+            origin = self.bounding_box.center
```

Comment:
perhaps I can try something like this which asks for a user specified origin when inf

```python
if origin is None:
    origin = self.bounding_box.center
    if np.inf in origin:
         raise ValueError("origin can't be found automatically and must be specified")
    origin = self.bounding_box.center
```

Reply:
To me, it seems a little annoying that running `universe.plot()` with no arguments would, in many cases, not work out of the box

Reply:
We can keep origin to default at (0,0,0) and scrap that part of the PR.

Perhaps one last idea 

```
if origin is None:
    origin = self.bounding_box.center
    if np.inf in origin:
         warning('Infinite value found when automatically attempting to set origin, setting infinite values to 0, manualy specify origin value to chose a different location')
         # setting origin to 0 when inf or -inf values found
         origin[~np.isfinite(origin)] = 0
    origin = self.bounding_box.center
```

Reply:
Even a warning here would be annoying. There's nothing inherently wrong with what the user did or their model. Again, I think we should just adopt the behavior that openmc-plotter does: if a bounding box can be found and the user hasn't said otherwise, use that bounding box to set the view. Otherwise, default to an origin of (0, 0, 0) and some arbitrary width. Right now the plotter uses (10, 10) by default.

Reply:
I've implemented this in the latest commit